tests/acceptance/new-test.js

```diff
@@ -625,7 +625,7 @@ describe('Acceptance: ember new', function () {
       checkFileWithEmberCLIVersionReplacement(fixturePath, 'tests/dummy/config/ember-cli-update.json');
     });
 
-    it('app + typescript', async function () {
+    it.skip('app + typescript', async function () {
```

Comment:
The intention was to skip them in node 14, not skip them in every node.

Reply:
I added a node check in the latest commit :partying_face: 

(I had tried checking NODE_VERSION on the env, but our test infra doesn't expose that, and I didn't think it made sense to mess with the test infra rn)

Reply:
![image](https://github.com/ember-cli/ember-cli/assets/199018/53d6f6ed-3b32-42c6-8b71-26a7dea776b6)


![image](https://github.com/ember-cli/ember-cli/assets/199018/d186532d-d5fa-4cee-b7a9-3c3d6403ae25)